.github/workflows/pr.yml

```diff
@@ -1,38 +1,88 @@
 name: Checks
 on: [push, pull_request]
+env:
+  NODE_VERSION: 20
```

Comment:
Before we had matrix strategy, but that got overly repetitive when splitting to multiple jobs. And we were not using the matrix functionality anyways as all jobs were run against a single node version.